src/twisted/web/test/test_http.py

```diff
@@ -2508,8 +2509,7 @@ def process(self):
 --AaBb1313--
 """
         channel = self.runRequest(req, MyRequest, success=False)
-        self.assertEqual(channel.transport.value(), b"HTTP/1.0 200 OK\r\n\r\ndone")
-        self.assertEqual(processed[0].args, {})
+        self.assertEqual(channel.transport.value(), b"HTTP/1.1 400 Bad Request\r\n\r\n")
```

Comment:
This looks like a backward incompatible change.

We can have this, no problem, but this needs to be part of the release notes.